src/main/scala/mesosphere/chaos/http/impl/HttpServiceImpl.scala

```diff
@@ -0,0 +1,74 @@
+package mesosphere.chaos.http.impl
+
+import java.util
+import javax.servlet.{Filter, Servlet, DispatcherType}
+
+import com.sun.jersey.api.core.DefaultResourceConfig
+import com.sun.jersey.spi.container.servlet.ServletContainer
+import org.eclipse.jetty.servlet.{FilterHolder, ServletHolder}
+import org.slf4j.LoggerFactory
+import mesosphere.chaos.http.HttpService
+
+import scala.util.Try
+/**
+  * Wrapper for starting and stopping the HttpServer.
+  */
+class HttpServiceImpl(httpServer: HttpServer) extends HttpService {
+  val servletPath = "/*"
+
+  private[this] val log = LoggerFactory.getLogger(getClass.getName)
+  private[this] var started = false
+
+  def startUp() {
+    log.debug("Starting up HttpServer.")
+    try {
+      started = true
+      preStartUp()
```

Comment:
Got that, but I also have a question on the `shutDown()` function: there is no `initialised = false` commando in there. In my opinion that is correct, as the `/*` servlet container, containing all registered resources, is already added to the servlet handler by this point. Calling the `preStartUp()` function as it is right now, would cause an error of adding a servlet to the same path. Do you agree on that?